apps/aecore/src/aec_coinbase_tx.erl

```diff
@@ -22,7 +22,7 @@
 -include("common.hrl").
 
 -record(coinbase_tx, {account       = <<>> :: pubkey(),
-                      block_height  = -1 :: integer()}).
```

Comment:
I believed you used the -1 as a placeholder for "undefined". Now your initial record has an undefined value, which is not an integer and also not a non_neg_integer(). This is only an issue if you ever use #coinbase_tx{} without setting the block_height field.

Reply:
Yes. We must not have coinbases without block height included.